resources/migrations/019_add_schema_column_to_table.yaml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+  - changeSet:
+      id: 19
+      author: camsaul
+      changes:
+        - addColumn:
+            tableName: metabase_table
+            columns:
+              - column:
+                  name: schema
+                  type: VARCHAR
```

Comment:
Is there a cross DB max size for schema?

Wouldn't want to limit it to 64 (say) and then raise it when we find out
SQL Server or Vertica allow 200 long chars?

On Mon, Nov 2, 2015 at 8:12 PM, Allen Gilliland <notifications@github.com>
wrote:

> In resources/migrations/019_add_schema_column_to_table.yaml
> <https://github.com/metabase/metabase/pull/1433#discussion_r43714836>:
>
> > @@ -0,0 +1,11 @@
> > +databaseChangeLog:
> > +  - changeSet:
> > +      id: 19
> > +      author: camsaul
> > +      changes:
> > +        - addColumn:
> > +            tableName: metabase_table
> > +            columns:
> > +              - column:
> > +                  name: schema
> > +                  type: VARCHAR
>
> Lets put character limit on this varchar. Having an unbound varchar
> doesn't play as nicely with databases other than postgres and realistically
> we don't need this to be unbound. From the postgres docs it looks like the
> max size for schema is 63 chars.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/metabase/metabase/pull/1433/files#r43714836>.
>